scripts-available/CDB_ColumnNames.sql

```diff
@@ -3,11 +3,12 @@ CREATE OR REPLACE FUNCTION CDB_ColumnNames(REGCLASS)
 RETURNS SETOF information_schema.sql_identifier
 AS $$
 
-    SELECT column_name 
-      FROM information_schema.columns 
-      WHERE
-        table_name IN (SELECT CDB_UserTables())
```

Comment:
this line seems to be a safeguard against peeking in tables in which you don't have permissions, just wondering why it was added...